### PR TITLE
Drop support for Django 4.1

### PR DESCRIPTION
**Pull request by francoisfreitag**

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -19,6 +19,7 @@ ChangeLog
 *Removed:*
 
 - Drop support for Django 4.0
+- Drop support for Django 4.1
```
Comment (rbarrois) [inline docs/changelog.rst:22]: I'd rephrase this as "Stop testing for Django 4.1 compatibility".
We're not actively dropping that support.

Comment (francoisfreitag) [reply docs/changelog.rst:22]: We’re removing the PyPI marker and no longer care for compatibility with Django 4.1, so I think drop support is accurate, although we’re not removing django 4.1 specific code?

**Review by kingbuzzman — APPROVED** [review]

File: docs/changelog.rst (hunk 1 above)
Comment (rbarrois) [reply docs/changelog.rst:22]: Well, when I read the changelog of a third-party package and see there "Drop support for Python 2", I understand that the code is likely to break with Python 2, for instance :)

Comment (francoisfreitag) [reply docs/changelog.rst]: Code could break for Django 4.1 and we wouldn’t know, or care. But enough discussion on this topic, I updated to a more precise phrasing.

Comment (jeffwidman) [reply docs/changelog.rst]: When I see the `"Drop support for X"` in other projects I tend to view it as "use at your own risk", it may or may not work... Just my $0.02.

What you've got is fine too.

**Review by jeffwidman — APPROVED** [review]

**Review by rbarrois — APPROVED** [review]
